package.json

```diff
@@ -68,10 +68,10 @@
     "@fortawesome/fontawesome-free": "5.7.2",
     "polished": "3.4.1",
     "react": "^16.8.0",
-    "react-dom": "^16.8.0",
     "react-debounce-input": "3.2.2",
-    "react-virtualized": "9.21.0",
```

Comment:
If I understand correctly, there is no change in `package.json`, only in `package-lock.json`
I am wondering what's the reason you wanted to commit the changes in package.json? Maybe write a sentence in the commit message?

Reply:
I committed this file by mistake